django-s3-file-field-client/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "django-s3-file-field-client",
```

Comment:
Could we call this simply `django-s3-file-field`, since I assume that name is not taken on NPM.